packages/synapse-constants/constants/tokens/index.ts

```diff
@@ -251,21 +255,27 @@ export const USD_POOLS_BY_CHAIN = getTokensByPoolTypeByChain('USD')
 export const ETH_POOLS_BY_CHAIN = getTokensByPoolTypeByChain('ETH')
 export const LEGACY_POOLS_BY_CHAIN = getLegacyTokensByChain()
 
-export const STAKABLE_TOKENS = {
+interface StakableTokens {
+  [chainId: string]: Token[]
+}
+
+export const STAKABLE_TOKENS: StakableTokens = {
   ...getPoolsByChain(false),
   [CHAINS.ETH.id]: [SYN_ETH_SUSHI_TOKEN],
 }
 
 const getStakingMap = () => {
-  const STAKING_MAP_TOKENS = {}
+  const STAKING_MAP_TOKENS: { [key: string]: any } = {}
```

Comment:
variable shadowing occurring on line 268 and 281, maybe good to update naming for one of them